### PR TITLE
Remove multiple final outputs in mpirun

### PR DESCRIPTION
**Pull request by siuwuncheung**

### CODE REVIEW

File: rom/laghos_rom.cpp [hunk 1]
```diff
@@ -2236,28 +2236,32 @@ void PrintNormsOfParGridFunctions(NormType normtype, const int rank, const std::
...
     double FOMnorm = sqrt(fomglob2);
     double ROMnorm = sqrt(romglob2);
     double DIFFnorm = sqrt(diffglob2);
     double relDIFFnorm = sqrt(diffglob2)/sqrt(fomglob2);
 
-    cout << rank << ": " << name << " FOM norm " << FOMnorm << endl;
-    cout << rank << ": " << name << " ROM norm " << ROMnorm << endl;
-    cout << rank << ": " << name << " DIFF norm " << DIFFnorm << endl;
-    cout << rank << ": " << name << " Rel. DIFF norm " << relDIFFnorm << endl;
+    if (rank == 0)
```
Comment (dylan-copeland) [inline rom/laghos_rom.cpp:2245]: The changes in this PR look fine, and it makes sense to print out only for rank 0, but this function seems wrong (before this PR). There is a TODO comment about some issues with this function, which we should resolve. It is not clear whether the MPI_Allreduce should be necessary in this function. It should be done in the `ParGridFunction` class. In any case, the L1 and max norms are clearly wrong (we typically use just L2).

**Review by kevinhkhuynh — APPROVED** [review]
I see Dylan made some comments. Just looking at the changes you've made, looks good to me.

File: rom/laghos_rom.cpp (hunk 1 above)
Comment (chldkdtn) [reply rom/laghos_rom.cpp:2245]: @dylan-copeland I will add it in "to-do-list" of our confluence page to look into this in the future (near future. :) )

**Review by chldkdtn — APPROVED** [review]

**Review by dylan-copeland — APPROVED** [review]
As noted above, this PR is fine, but we should keep in mind that the parallel norm issue should be investigated.

**Pull request merged**